roles/init_dbserver/tasks/pg_initdb.yml

```diff
@@ -49,3 +49,17 @@
     state: started
     enabled: true
   become: true
+
+- name: Gather service facts
+  service_facts:
+
+- name: Start pg service if not running
+  systemd:
+    name: "{{ pg_service }}"
+    state: started
+    daemon_reload: true
+    enabled: true
+  when:
+    - ansible_facts.services[pg_service] is defined
+    - ansible_facts.services[pg_service].state == 'running'
+  become: yes
```

Comment:
Is it required? Postgres service must have been started by the previous task.

Reply:
I did that because with `force_initdb: true` ansible wasn't successful